Cython/Compiler/Errors.py

```diff
@@ -54,7 +56,7 @@ def format_error(message, position):
     if position:
         pos_str = format_position(position)
         cont = context(position)
-        message = '\nError compiling Cython file:\n%s\n%s%s' % (cont, pos_str, message or '')
+        message = '\nError compiling Cython file:\n%s%s%s' % (cont, pos_str, message or '')
```

Comment:
It surprised me that this was necessary. It removes the blank line between the code context block and the error message line in the error report output. I'm not sure where that line came from or why it was there, it seems distracting to me.